database/index.js

```diff
@@ -1,8 +1,16 @@
-//dev2
-
 const mongoose = require('mongoose');
+require('dotenv').config();
 //const uri = 'mongodb://mongo:27017/reviewsDB';
-const uri = 'mongodb://localhost/reviewsDB';
+//const uri = 'mongodb://localhost/reviewsDB';
+//const mode = process.env.NODE_ENV;
+let uri = '';
+
+if (process.env.NODE_ENV === "production") {
+  uri = 'mongodb://mongo:27017/reviewsDB'
+} else {
+  uri = 'mongodb://localhost/reviewsDB'
+}
+console.log('uri', uri, 'prod', process.env.NODE_ENV)
```

Comment:
nice logic! i suggest some code clean up here (delete the lines you no longer need)